app/javascript/components/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Provider } from 'react-redux';
+import {  BrowserRouter, Routes, Route } from 'react-router-dom';
+import Greeting from './Greeting';
+import Header from './Header';
```

Comment:
- [ ] Here you have named `Header` but your file name is `header`. Please fix it, otherwise, your application giving an error.